templates/form-fields/file-field.php

```diff
@@ -4,7 +4,7 @@
 $field_name         = isset( $field['name'] ) ? $field['name'] : $key;
 $field_name         .= ! empty( $field['multiple'] ) ? '[]' : '';
 
-if ( ! empty( $field['ajax'] ) ) {
+if ( ! empty( $field['ajax'] ) && job_manager_user_can_upload_file_via_ajax() ) {
```

Comment:
Only comment I have is that if theme devs and myself update our template overrides to include this, if a user installs or is using an older version of WPJM it will throw fatal error, so we will have to check if function exists first before calling, besides that looks good to me